styleguide/derek/view-templates/stories/_stories.scss

```diff
@@ -1,3 +1,218 @@
+// Customer Stories Feature Colors
+$cust-stories-red: #9d0000;
+$cust-stories-teal: #00b4c3;
+$cust-stories-purple: #563777;
```

Comment:
@amberRrucker we could with the red and teal but not the purple, it doesn't exist